src/parse/request.js

```diff
@@ -53,6 +56,7 @@ function addBoundary(boundary, headers) {
       return item
     })
 
+    headers.delete('Content-Type') // Remove uppercased content-type in case it exists
```

Comment:
I cannot help but feel that this should be part of some kind of general sanitizing/normalization phase. Feels like we are doing this in other places as well?

Reply:
You're right, there are some inconsistencies between how the headers are being set (lower-cased or upper-cased). In this case that caused a bug under certain circumstances.

Reply:
Agree with @allansson , there should be normalization of the incoming headers in the parse stage really. Although I think this can be a future improvement for the sake of getting this one out 👍

Reply:
I agree. Was just something I thought about when I saw it. :)